### ISSUE TITLE
Increased Defecation/ non-regular bowel movements

### ISSUE THREAD

**Issue by Lucy-WhiteSwan** [obophenotype/human-phenotype-ontology#7517]
**Preferred term label:**
Increased Defecation

**Synonyms**
Pooing more often, bowel movements more often

**Definition (free text, please give PubMed ID)**
PubMed ID: N/A, non-regular bowel movements

**Parent term (use hpo.jax.org/app)**
Abnormality of the digestive system HP:0025031

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:443909

**Your nano-attribution (ORCID)**
n/a

**Comment by pnrobinson** [comment]
@Lucy-WhiteSwan  it would be good to delineate this from diarrhea.
Diarrhea can be defined as three or more loose or watery stools a day (PMID:28846339)
Increased defecation does not seem the same as non-regular bowel movements, but a glance at some websites suggests that many use terms such as 'frequent bowel movements" and "frequent bowel movements" as synonyms for diarrhea. Can you provide more information?

**Comment by Lucy-WhiteSwan** [comment]
Hi @pnrobinson, apologies for the delayed reply, if patients' bowel movements became more regular or often, but are not bad/symptomatic enough to be defined as 'diarrhea', then (from pubmed), it looks like the usual description is 'Change in bowel habit'. I think 'more frequent bowel habit', and 'less frequent bowel habit' would be decedents of 'change in bowel habit', but there isn't a clear example on pubmed for those.

Here are some examples of the phrase/symptom 'Change in bowel habit' on pubmed:
PMID: 32213069
PMID: 30898041
PMID: 17217445

**Comment by pnrobinson** [comment]
@Lucy-WhiteSwan I see what you mean. I will try to find a definition for these terms, since I would hazard to guess that people might define these terms differently unless we are very precise.

**Comment by pnrobinson** [comment]
Sorry for the late response -- multiple reasons including relocation to Germany.
I am not able to find precise definitions for these terms and suspect they are likely to be used in an inconsistent way that would defeat the point. I would suggest using the existing term [Diarrhea](https://hpo.jax.org/app/browse/term/HP:0002014) is warranted. If there is a society definition or some other "official" source, please reopen this request.
